ostd/src/mm/page/meta.rs

```diff
@@ -209,9 +241,23 @@ fn alloc_meta_pages(nframes: usize) -> Paddr {
         .alloc(nframes)
         .unwrap()
         * PAGE_SIZE;
-    // Zero them out as initialization.
+
+    const INIT_BYTE: u8 = 0xff;
+
+    // Assert that the byte pattern represents an unused `MetaSlot`.
+    const _: () = const {
+        let mut meta_slot = MaybeUninit::<MetaSlot>::uninit();
+        unsafe { meta_slot.as_mut_ptr().write_bytes(INIT_BYTE, 1) };
+        let meta_slot = unsafe { meta_slot.assume_init() };
+        assert!(meta_slot.ref_count.into_inner() == REF_COUNT_UNUSED);
```

Comment:
@junyang-zh

#1724 seems to add another `is_untyped: UnsafeCell<bool>` (we should use `AtomicBool` instead IMO, but that's another topic), which makes the byte pattern here invalid again, do you have any comments on this fact?

We can use one of the following approaches:
 - Use `is_untyped: UnsafeCell<MaybeUninit<bool>>` in #1724.
 - Consider using a different byte pattern to fill the meta pages. However, the only possible byte pattern is "all zeros", which is not really good because a zero in the reference count cannot mean `REF_COUNT_UNUSED`.

Reply:
I guess efficiency is not a concern for initializing the meta pages so we can fill them one by one instead of using a byte pattern?

Reply:
`UnsafeCell<MaybeUninit<bool>>` looks good too. And I think it doesn't involve synchronization since we only sync on the reference count. Why should it be a `AtomicBool` btw?

Reply:
> I guess efficiency is not a concern for initializing the meta pages so we can fill them one by one instead of using a byte pattern?

Makes sense. This may affect boot time, but the impact shouldn't be much. Let's change it to fill one at a time to see if there is any observed performance difference.